src/servers/main.cc

```diff
@@ -308,6 +309,9 @@ std::vector<Option> options_
        "request/response objects."},
       {OPTION_GRPC_USE_SSL, "grpc-use-ssl", Option::ArgBool,
        "Use SSL authentication for GRPC requests. Default is false."},
+      {OPTION_GRPC_USE_SSL_MUTUAL, "grpc-use-ssl-mutual", Option::ArgBool,
+       "Use mututal SSL authentication for GRPC requests. Default is false. If "
+       "true, set --grpc-use-ssl to true."},
```

Comment:
I would remove the "If true..." sentence. Internally we will set options as necessary.